hw/bsp/nrf52dk/pkg.yml

```diff
@@ -39,8 +39,8 @@ pkg.cflags:
     - '-DPDM_ENABLED=0'
     - '-DPERIPHERAL_RESOURCE_SHARING_ENABLED=1'
     - '-DPWM0_ENABLED=1'
-    - '-DPWM1_ENABLED=0'
-    - '-DPWM2_ENABLED=0'
+    - '-DPWM1_ENABLED=1'
+    - '-DPWM2_ENABLED=1'
```

Comment:
@mlaz The default bsp in mynewt does not need to enable all PWMs. One is enabled and we can keep it at that to be used as an example. Any specific reason why you want to enable all by default ?